resources/service-catalog/charts/etcd-stateful/templates/05-backup-job.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           serviceAccountName: {{ template "etcd-fullname" . }}-backup
           containers:
           - name: etcd-backup
-            image: "{{ .Values.global.containerRegistry.path }}/{{ .Values.global.etcd_backup.dir }}etcd-backup:{{ .Values.global.etcd_backup.version }}"
+            image: "{{ .Values.global.containerRegistry.path }}/{{ .Values.global.etcd_backup_job.dir }}etcd-backup:{{ .Values.global.etcd_backup.version }}"
```

Comment:
missed version part, should be `.Values.global.etcd_backup_job.version` is `.Values.global.etcd_backup.version`

Reply:
good catch, fixed